src/main/scala/com/fulcrumgenomics/cmdline/ClpGroups.scala

```diff
@@ -58,11 +58,17 @@ object ClpGroups {
     override def description: String = "Tools for manipulating UMIs & reads tagged with UMIs"
   }
 
+  class _PhasingGroup extends ClpGroup {
+    override def name: String = "Variant Phasing"
+    override def description: String = "Tools for working and producing with phased variants"
+  }
+
   final val Personal  = classOf[_PersonalGroup]
   final val SamOrBam  = classOf[_SamOrBamGroup]
   final val Fasta     = classOf[_FastaGroup]
   final val Fastq     = classOf[_FastqGroup]
   final val Umi       = classOf[_Umi]
   final val Utilities = classOf[_UtilitiesGroup]
+  final val Phasing   = classOf[_PhasingGroup]
```

Comment:
Are we likely to have > 1 tool for phasing over time?  If not, perhaps just a variants group?